tests/resources/options_continue.yaml

```diff
@@ -0,0 +1,18 @@
+architecture:
+  name: soap_bpnn
+  model:
+    restart: bpnn-model.pt
+  training:
+    batch_size: 2
+    num_epochs: 1
+
+# Section defining the parameters for structure and target data
+training_set:
+  structures:
+    read_from: "qm9_reduced_100.xyz"
+  targets:
+    energy:
+      key: "U0"
+
+test_set: 0.1
+validation_set: 0.1
```

Comment:
I would rather try to overwrite the options in the test instead of adding more files. Basically works like this:

```python
    options = OmegaConf.load(RESOURCES_PATH / "options.yaml")
    options["foo"] = "bar"
    OmegaConf.save(config=options, f="options.yaml")
```

Reply:
Ok!